helix-term/src/ui/menu.rs

```diff
@@ -23,7 +23,7 @@ pub trait Item {
         self.label()
     }
 
-    fn row(&self) -> Row {
+    fn row(&self, _theme: Option<&Theme>) -> Row {
```

Comment:
Why do we want this to be `Option`? I think it might be better to just always pass it in since we know we have it.

Reply:
It's not about having it, it's about skipping the style since it won't be used in those instances. This could be a small performance improvement but could be good in massive datasets. Especially skipping the `try_get(style)` function.